shell.nix

```
@@ -21,13 +22,39 @@ shared.pkgs.runCommand "dhall-shell-${shell}" {
     echo "    Overriding option completion shell flavor:"
     echo "        $ nix-shell --argstr shell zsh"
 
-    source <(dhall            --${shell}-completion-script dhall)
-    source <(dhall-to-bash    --${shell}-completion-script dhall-to-bash)
-    source <(dhall-to-nix     --${shell}-completion-script dhall-to-nix)
-    source <(dhall-to-json    --${shell}-completion-script dhall-to-json)
-    source <(dhall-to-yaml    --${shell}-completion-script dhall-to-yaml)
-    source <(dhall-to-yaml-ng --${shell}-completion-script dhall-to-yaml-ng)
-    source <(json-to-dhall    --${shell}-completion-script json-to-dhall)
-    source <(yaml-to-dhall    --${shell}-completion-script yaml-to-dhall)
+    EXECUTABLES=(
+      dhall
```

Not sure whether it's too early?!

```suggestion
      dhall
      dhall-docs
```